### ISSUE TITLE
WCAG 1.3.1: Ensures <dl> elements are structured correctly (dl)

### ISSUE THREAD

**Issue by oklimanova** [oklimanova/hello-world#22]
#### Issue
`Ensures <dl> elements are structured correctly` ([definition-list](https://dequeuniversity.com/rules/axe/4.0/definition-list?application=msftAI))
#### Target application
[ROCIS - Rule Search](http://192.168.1.251:8080/rocis/SearchRule.do)
#### Element path
td[width="\32 0\%"] > dl
#### Snippet
`<dl>`
#### How to fix
    Fix all of the following:
      When not empty, element does not have at least one <dt> element followed by at least one <dd> element
      List element has direct children that are not allowed inside <dt> or <dd> elements
#### Environment
Chrome version 85.0.4183.83
This accessibility issue was found using Accessibility Insights for Web 2.21.1 (axe-core 4.0.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
